src/transport/node/builders/message.ts

```
@@ -1,6 +1,8 @@
 import type { WaButtonAddonKind } from '@message/encode/content'
-import { WA_MESSAGE_TAGS, WA_MESSAGE_TYPES, WA_NODE_TAGS } from '@protocol/constants'
 import type { BinaryNode } from '@transport/types'
+import { WA_MESSAGE_TAGS, WA_MESSAGE_TYPES, WA_NODE_TAGS } from '@protocol/constants'
```

ESLint

[error] `@protocol/constants` import should occur before type import of `@transport/types`

(import/order)